templates/local.conf.j2

```diff
@@ -0,0 +1,16 @@
+[Time]
+{% if ntp_servers is defined %}
+NTP={{ ntp_servers | join(" ") }}
+{% else %}
```

Comment:
For a drop-in file, we don't need the `{% else %}` stuff. This file should not have commented out config lines.